guide/src/index.html

```diff
@@ -1,7 +1,7 @@
 <html>
     <head>
         <meta charset="utf-8" />
-        <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css' />
+        <link href='https://fonts.googleapis.com/css?family=Roboto:300,400' rel='stylesheet' type='text/css' />
```

Comment:
I just know in productions, Troposphere links to Roboto like so:
```
<link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet" type="text/css">
```

I'm not sure the impact, I was just mentioning what I thought I saw. I don't know if has caused any issues.